src/screens/ContactScreen/elements/SearchRecyclerView.js

```diff
@@ -150,7 +150,13 @@ const SearchRecyclerView = ({text, onHandleSelected, followed, setLoading}) => {
         }}
         rowRenderer={rowRenderer}
         scrollViewProps={{
-          refreshControl: <RefreshControl refreshing={refreshing} onRefresh={() => {}} />
+          refreshControl: (
+            <RefreshControl
+              tintColor={COLORS.white2}
+              refreshing={refreshing}
+              onRefresh={() => {}}
+            />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `RefreshControl` component has been updated with a new `tintColor` prop, but the `onRefresh` function is still an empty function. This means that when users pull to refresh, nothing will happen. It's important to provide a meaningful function for `onRefresh` to improve user experience and functionality of the app.

```diff
- onRefresh={() => {}}
+ onRefresh={this.handleRefresh} // Assuming handleRefresh is a method defined in your component
```
Please ensure that you have a function that handles the refresh logic.


<!-- This is an auto-generated comment by OSS CodeRabbit -->